rudder-core/src/main/scala/com/normation/rudder/services/policies/write/BuildBundleSequence.scala

```diff
@@ -146,8 +152,10 @@ object BuildBundleSequence {
     , isSystem               : Boolean
     , providesExpectedReports: Boolean
     , policyMode             : PolicyMode
+    , bundleContext          : Option[BundleContext]
```

Comment:
Why an option ? We always have that information. Is there some case where we don't want to use it?

Reply:
I don't think we want to define context for all Bundles ( We only want it for the main Bundle, not the pre and the post ones) 

But maybe we could add the bundle in the right place without having to set this ... I'll look if i find a better way